plugins/module_utils/ece.py

```diff
@@ -109,17 +109,14 @@ def get_snapshot_repo_by_name(self, repo_name):
     repos = self.get_snapshot_repos()
     return next(filter(lambda x: x['repository_name'] == repo_name, repos), None)
 
-  def get_deployment_id(self,deployment_name):
+  def get_deployment_kibana_info(self,deployment_name):
+      deployment_object = ''
```

Comment:
Is there a reason to use an empty string rather than None here?

Reply:
Just to establish it is a string.  Does it make a difference?